packages/manager/modules/vrack/src/vrack-associated-services/ipv6/translations/Messages_fr_FR.json

```diff
@@ -11,9 +11,13 @@
   "vrack__assigned-ipv6__subnet_delete_title": "Retirer le sous réseau de votre vRack",
   "vrack__assigned-ipv6__subnet_delete_description": "Confirmez-vous vouloir retirer le sous réseau de votre vRack ?",
   "vrack__assigned-ipv6_subnet_via": "{{ ipblock }} via {{ ip }} ",
+  "vrack_add_ipv6_modal_title": "Ajouter un IP block dans le vRack",
+  "vrack_add_ipv6_modal_description": "Vous allez ajouter un bloc IP : <strong>{{ ipblock }}</strong> (situé dans <strong>{{ region }}</strong>) dans votre vRack : <strong>{{ vrack }}</strong> <br> Cette action permettra d'atteindre les machines situées dans ce vRack depuis l'internet public.<br>Le trafic public sera routé à travers la région <strong>{{ region }}</strong>.<br>Pour plus d'information, veuillez consulter",
```

Comment:
**nitpick:** Avoid break lines in translations by splitting into multiple messages.